### ISSUE TITLE
Color picker should be able to be used via a keyboard

### ISSUE THREAD

**Issue by bmattb** [GoogleForCreators/web-stories-wp#8918]
## Feature Description

Users attempting to use the color picker with only a keyboard are unable to do so. This could be considered an accessibility issue for vision impaired users who are not able to use a mouse.

**Comment by swissspidy** [comment]
Let's not make accessibility an afterthought next time :-)

**Comment by barklund** [comment]
@swissspidy, is this before or after #8742 was merged, do you know?

**Comment by swissspidy** [comment]
I didn't originally report this, so IDK. #8742 was merged before bug bash though, so I assume _after_.
But @bmattb would need to confirm

**Comment by miina** [comment]
@barklund Is this still an issue? IIRC you were working on the Color Picker accessibility at some point.

**Comment by miina** [comment]
Closing since this has been implemented meanwhile.
